tests/EnergonTest.php

```diff
@@ -105,4 +106,13 @@ public function testInvalidPrimeProvided()
 
         Energon::generate(2);
     }
+
+    public function testSetPrimeShouldThrowInvalidPrimeException()
+    {
+        $this->setExpectedException('Jenssegers\Optimus\Exceptions\InvalidPrimeException', '20');
+
```

Comment:
Could you use ::class here?

Reply:
I use the string instance of ```::class``` because I refer the previous approach.

Reply:
Unfortunately, the version ```5.4``` should support the ```::class``` and it will support this since the version ```5.5```.
Please refer the [doc](http://php.net/manual/en/language.oop5.constants.php) in ```Example #3```.

Reply:
I'm ok with increasing the minimum version to 5.5